apps/parsley/src/components/SectionsFeatureModal/index.tsx

```diff
@@ -0,0 +1,65 @@
+import styled from "@emotion/styled";
+import MarketingModal from "@leafygreen-ui/marketing-modal";
+import { useMatch } from "react-router-dom";
+import { useSectionsFeatureDiscoveryAnalytics } from "analytics/sectionsFeatureDiscovery/useSectionsFeatureDiscoveryAnalytics";
+import routes from "constants/routes";
+import { zIndex } from "constants/tokens";
+import { useSectionsFeatureDiscoveryContext } from "context/SectionsFeatureDiscoveryContext";
+import { useParsleySettings } from "hooks/useParsleySettings";
+import { releaseSectioning } from "utils/featureFlag";
+import { graphic } from "./graphic";
+
+export const SectionsFeatureModal = () => {
```

Comment:
We should avoid adding this component and large graphic into the bundle and instead only lazy load it if we plan on showing it.